crd-generator/test/src/test/java/io/fabric8/crd/generator/approvaltests/CRDGeneratorApprovalTest.java

```diff
@@ -148,6 +152,14 @@ static Stream<TestCase> crdApprovalBaseCases(String crdVersion) {
     return cases.stream();
   }
 
+  static Stream<TestCase> crdApprovalCasesApiV2(String crdVersion) {
```

Comment:
@manusa I have started here with splitting test cases into two groups:  

- The existing tests for api-v1 and api-v2.
- Test cases targeting new features in api-v2 only.


I think this change is also important for other feature additions. Please let me know if I should prepare this in an extra PR to be able to merge this faster.

Reply:
see https://github.com/fabric8io/kubernetes-client/pull/6533